src/test/java/nl/dtls/fairdatapoint/api/controller/MetadataControllerTest.java

```diff
@@ -321,6 +353,48 @@ public void getContentWithFileExtRepo() throws Exception {
         assertEquals("text/turtle", response.getContentType());
     }
     
+    /**
+     * Check url ends with /
+     *
+     * @throws Exception
+     */
+    @DirtiesContext
+    @Test
+    public void checkURLFilter() throws Exception {
```

Comment:
Same remark as above about the need for `@DirtiesContext`.

Reply:
Same response as a previous comment